epictrack-api/src/api/models/work_issues.py

```diff
@@ -36,5 +39,14 @@ class WorkIssues(BaseModelVersioned):
     work_id = Column(ForeignKey('works.id'), nullable=False)
     work = relationship('Work', foreign_keys=[work_id], lazy='select')
 
-    # Define a relationship to WorkIssueUpdates
     updates = relationship('WorkIssueUpdates', back_populates='work_issue', lazy='joined')
+
+    @classmethod
+    def list_issues_for_work_id(cls, work_id) -> List[WorkIssues]:
+        """List all WorkIssues sorted by start_date, with approved issues first."""
+        query = (
+            WorkIssues.query
+            .filter(cls.work_id == work_id)
+            .order_by(cls.is_approved.desc(), cls.start_date)
```

Comment:
Grabbing your attention to this line, will this have the latest start date at the first index or the oldest? We might need to discuss to decide